src/platforms/javascript/common/session-replay/configuration.mdx

```diff
@@ -87,6 +88,50 @@ new Replay({
 
 <Note>Captured bodies will be truncated to 150k characters max.</Note>
 
+## Identifying Users
```

Comment:
Moved this up above "Mutation limits" as IMHO mutation limits is the least important config described on this page...?